content/en/docs/tutorials/clusters/seccomp.md

```diff
@@ -45,6 +45,38 @@ for the version you are using.
 
 <!-- steps -->
 
+## Enable the `RuntimeDefault` seccomp profile for all workloads
+
+A new kubelet alpha [feature gate](/docs/reference/command-line-tools-reference/feature-gates)
+`SeccompDefault` has been introduced with the release of Kubernetes v1.22.0.
```

Comment:
This should use a [feature state shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state).

Reply:
Added a feature short code for this section

Reply:
With the feature state line in place, you can omit this one. It should come out because phrases like this are relevant at the point of release but don't age well, whereas a feature-state marker is valid until the facts change.

(This is different from blog articles which absolutely can and do use wording similar to the line I'm commenting on).